common/protos/proto/config.proto

```diff
@@ -12,6 +12,16 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-mod file_s3;
+syntax = "proto3";
 
-pub use file_s3::S3File;
+package databend_proto;
+
+message S3StorageConfig {
+  string region = 1;
+  string endpoint_url = 2;
```

Comment:
There should be a `ver` to identify the `proto-conv` version if this message has chances to be used alone. Otherwise, without a `ver`, it implies `proto-conv` will always have to be able to deal with any old version `S3StorageConfig` message.

Reply:
I agree with your concern.

Configs will have their own version flags (our binary also relies upon the same struct). I'm going to add them in another PRs.

Reply:
Tracked at https://github.com/datafuselabs/databend/issues/5297

Reply:
@Xuanwo  I think it would be nice to do it in this PR, or it would be a compatibility issue again.

Reply:
I'm using the default `VER` for now and will have a complete solution after #5297 is finished.